drf_braces/tests/serializers/test_form_serialzer.py

```diff
@@ -6,7 +6,7 @@
 import mock
```

Comment:
thanks for the typo fix rename. although there is still a typo

although its still missing second `i` in `serializer`...

can you fix that one as well?

Reply:
👍